beast/fitting/fit_metrics/kernels.py

```diff
@@ -123,7 +124,7 @@ def __init__(self, shape, h = 1.0, domain = None, norm = None):
             norm = 1.0
         self._normconst = norm
         self.domain = domain
-        if callable(shape):
+        if isinstance(shape, collections.Callable):
```

Comment:
Caution, I've never used that object but  it refers to an ABC class that has multiple methods.
I do not know if any object with a __call__() method enters that category.
The callable function was simply a test on the __call__ method

class collections.Callable
ABCs for classes that provide respectively the methods __contains__(), __hash__(), __len__(), and __call__().